samples/tab-calendar/ts/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-tab-calendar-ts",
+    "source": "officeDev",
+    "title": "Tab calendar in Outlook",
+    "shortDescription": "Create React App personal tab in TypeScript",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/tab-calendar/ts",
+    "longDescription": [
+      "This sample app shows interact with the user's calendar, including opening calendar items and composing meetings."
+    ],
+    "creationDateTime": "2023-06-19",
+    "updateDateTime": "2023-06-10",
+    "products": [
+      "Teams"
+    ],
+    "metadata": [
+      {
+        "key": "TEAMS-SAMPLE-SOURCE",
+        "value": "OfficeDev"
+      },
+      {
+        "key": "TEAMS-SERVER-LANGUAGE",
+        "value": ""
+      },
+      {
+        "key": "TEAMS-SERVER-PLATFORM",
+        "value": ""
+      },
+      {
+        "key": "TEAMS-FEATURES",
+        "value": "tab"
+      }
+    ],
+    "thumbnails": [
+      {
+        "type": "image",
+        "order": 100,
+        "url": "https://raw.githubusercontent.com/OfficeDev/Microsoft-Teams-Samples/main/samples/tab-calendar/ts/Images/TabPersonalQuickstartGif.gif",
+        "alt": "Solution UX showing Tab calendar support in Outlook apps."
```

Comment:
Solution UX showing how to interact with Calendar in Outlook from Personal Tab.

Reply:
Fixed